test/libsyncengine/update_detection/file_system_observer/testcomputefsoperationworker.cpp

```diff
@@ -243,4 +265,82 @@ void TestComputeFSOperationWorker::testLnkFileAlreadySynchronized() {
+    _syncPal->syncDb()->insertNode(nodeTest, dbNodeIdTest, constraintError);
+
+    _syncPal->_localSnapshot->updateItem(SnapshotItem("ltest", *_syncPal->syncDb()->rootNode().nodeIdLocal(), Str("testé.txt"),
+                                                      testhelpers::defaultTime, testhelpers::defaultTime, NodeType::File,
+                                                      testhelpers::defaultFileSize));
+
+    _syncPal->copySnapshots();
+    _syncPal->_computeFSOperationsWorker->execute();
+    FSOpPtr tmpOp = nullptr;
+    CPPUNIT_ASSERT(_syncPal->_localOperationSet->findOp("ltest", OperationType::Move, tmpOp));
+}
+
+void TestComputeFSOperationWorker::testDifferentEncoding_NFD_NFD() {
+    // NFD in DB, NFD on FS
+    DbNode nodeTest(0, _syncPal->syncDb()->rootNode().nodeId(), Str("testé.txt"), Str("testé.txt"), "ltest", "rtest",
+                    testhelpers::defaultTime, testhelpers::defaultTime, testhelpers::defaultTime, NodeType::File,
+                    testhelpers::defaultFileSize, std::nullopt);
+    DbNodeId dbNodeIdTest;
+    bool constraintError = false;
+    _syncPal->syncDb()->insertNode(nodeTest, dbNodeIdTest, constraintError);
+
+    _syncPal->_localSnapshot->updateItem(SnapshotItem("ltest", *_syncPal->syncDb()->rootNode().nodeIdLocal(), Str("testé.txt"),
+                                                      testhelpers::defaultTime, testhelpers::defaultTime, NodeType::File,
+                                                      testhelpers::defaultFileSize));
+
+    _syncPal->copySnapshots();
+    _syncPal->_computeFSOperationsWorker->execute();
+    FSOpPtr tmpOp = nullptr;
+    CPPUNIT_ASSERT(!_syncPal->_localOperationSet->findOp("ltest", OperationType::Move, tmpOp));
+    CPPUNIT_ASSERT(_syncPal->_localOperationSet->nbOps() == 0);
+}
+
+void TestComputeFSOperationWorker::testDifferentEncoding_NFC_NFC() {
+    // NFC in DB, NFC on FS
+    DbNode nodeTest(0, _syncPal->syncDb()->rootNode().nodeId(), Str("testé.txt"), Str("testé.txt"), "ltest", "rtest",
+                    testhelpers::defaultTime, testhelpers::defaultTime, testhelpers::defaultTime, NodeType::File,
+                    testhelpers::defaultFileSize, std::nullopt);
+    DbNodeId dbNodeIdTest;
+    bool constraintError = false;
+    _syncPal->syncDb()->insertNode(nodeTest, dbNodeIdTest, constraintError);
+
+    _syncPal->_localSnapshot->updateItem(SnapshotItem("ltest", *_syncPal->syncDb()->rootNode().nodeIdLocal(), Str("testé.txt"),
+                                                      testhelpers::defaultTime, testhelpers::defaultTime, NodeType::File,
+                                                      testhelpers::defaultFileSize));
+
+    _syncPal->copySnapshots();
+    _syncPal->_computeFSOperationsWorker->execute();
+    FSOpPtr tmpOp = nullptr;
+    CPPUNIT_ASSERT(!_syncPal->_localOperationSet->findOp("ltest", OperationType::Move, tmpOp));
+    CPPUNIT_ASSERT(_syncPal->_localOperationSet->nbOps() == 0);
```

Comment:
Public accessor for `_localOperationSet`?

Reply:
Done